Add plugin: Show Whitespace

ebullient opened the pull request:
# I am submitting a new Community Plugin

## Repo URL

<!--- Paste a link to your repo here for easy access -->
Link to my plugin:

## Release Checklist
- [x] I have tested the plugin on
  - [ ]  Windows
  - [x]  macOS
  - [x]  Linux
  - [ ]  Android _(if applicable)_
  - [x]  iOS _(if applicable)_
- [x] My GitHub release contains all required files
  - [x] `main.js`
  - [x] `manifest.json`
  - [x] `styles.css` _(optional)_
- [x] GitHub release name matches the exact version number specified in my manifest.json (_**Note:** Use the exact version number, don't include a prefix `v`_)
- [x] The `id` in my `manifest.json` matches the `id` in the `community-plugins.json` file.
- [x] My README.md describes the plugin's purpose and provides clear usage instructions.
- [x] I have read the tips in https://docs.obsidian.md/Plugins/Releasing/Plugin+guidelines and have self-reviewed my plugin to avoid these common pitfalls.
- [x] I have added a license in the LICENSE file.
- [x] My project respects and is compatible with the original license of any code from other plugins that I'm using.
      I have given proper attribution to these other projects in my `README.md`.

joethei commented:
["id": "show-whitespace-cm6",](https://github.com/ebullient/obsidian-show-whitespace-cm6/blob/6f8656a5ee984c99cdc6d74a31e3b87a85f8b033/manifest.json#L2-L3), ["description": "Show leading and trailing whitespace in source mode (CodeMirror6)",](https://github.com/ebullient/obsidian-show-whitespace-cm6/blob/6f8656a5ee984c99cdc6d74a31e3b87a85f8b033/manifest.json#L6)
You can remove all references to CM6 here, I just removed the original plugin from the list.

ebullient commented:
> ["id": "show-whitespace-cm6",](https://github.com/ebullient/obsidian-show-whitespace-cm6/blob/6f8656a5ee984c99cdc6d74a31e3b87a85f8b033/manifest.json#L2-L3), ["description": "Show leading and trailing whitespace in source mode (CodeMirror6)",](https://github.com/ebullient/obsidian-show-whitespace-cm6/blob/6f8656a5ee984c99cdc6d74a31e3b87a85f8b033/manifest.json#L6) You can remove all references to CM6 here, I just removed the original plugin from the list.

I will leave repository references to CM6 (invasive, and who knows.. someday there will be CM7), but I've removed from UI-visible name.
